attestation-agent/attester/Cargo.toml

```
@@ -26,6 +26,8 @@ codicon = { version = "3.0", optional = true }
 hyper = { version = "0.14", features = ["full"], optional = true }
 hyper-tls = { version = "0.5", optional = true }
 tokio = { version = "1", features = ["full"], optional = true }
+tempfile = { version = "3.9.0", optional = true }
```

We can update `tempfile` in the top `Cargo.toml` and refer it here
```suggestion
tempfile = { workspace = true, optional = true }
```